spec/fixtures/custom_nginx.template

```diff
@@ -90,11 +86,7 @@ http {
         client_body_buffer_size ${{CLIENT_BODY_BUFFER_SIZE}};
 
 > if ssl then
-> if real_ip_header == "proxy_protocol" then
-        listen ${{PROXY_LISTEN_SSL}} proxy_protocol ssl;
-> else
-        listen ${{PROXY_LISTEN_SSL}} ssl;
-> end
+        listen ${{PROXY_LISTEN_SSL}} ssl${{PROXY_PROTOCOL}};
```

Comment:
Interesting that the equivalent line from the actual template is not the same as this one, they somehow diverged and the `${{HTTP2}}` variable was not included:
```
listen ${{PROXY_LISTEN_SSL}} ssl${{HTTP2}}${{PROXY_PROTOCOL}};
```